POST.md

```diff
@@ -0,0 +1,253 @@
+# @platformatic/kafka Performance Evolution: A Journey to v1.21.0
+
+In our [previous blog post](https://blog.platformatic.dev/why-we-created-another-kafka-client-for-nodejs), we introduced `@platformatic/kafka` and shared performance benchmarks comparing it against popular Node.js Kafka clients like KafkaJS and node-rdkafka. Since then, we've continued improving the library, and the results speak for themselves.
+
+Over the past several releases leading to v1.21.0, we achieved **dramatic performance improvements** without changing a single line of benchmark code. How? By optimizing the core library implementation and fixing critical bugs that were holding back performance.
+
+## Performance-Critical Improvements
```

Comment:
I think there were a few other PRs from external collaborators that helped as well.